Resources/Textures/Objects/Devices/health_analyzer.rsi/meta.json

```diff
@@ -0,0 +1,26 @@
+{
+  "version": 1,
+  "size": {
+    "x": 32,
+    "y": 32
+  },
+  "license": "CC-BY-SA-3.0",
+  "copyright": "Taken from tgstation",
```

Comment:
This needs to reference the commit they were taken from.

Reply:
I got the images from this project(several different pull requests). I have to set a link for every pull request?

Reply:
Just the commit you looked at their repo.